sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 1.9.0-beta.1 (Unreleased)
```

Comment:
I am assuming we should close this PR and the increment PR after the beta release https://github.com/Azure/azure-sdk-for-cpp/pull/4516 is merged is the one we want.